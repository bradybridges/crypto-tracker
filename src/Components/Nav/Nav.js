import React from 'react';
import { NavLink } from 'react-router-dom';
import './Nav.scss';
import searchImg from '../../Images/search.svg';
import homeImg from '../../Images/home.svg';
import portfolioImg from '../../Images/portfolio.svg';

const Nav = () => {
  return (
    <>
      <NavLink className='nav-button' to='/'>
        <img src={homeImg} alt='home button' />
      </NavLink>
      <NavLink className='nav-button' to='/'>
        <img src={portfolioImg} alt='portfolio button' />
      </NavLink>
      <NavLink to='/hello' className='nav-button'>
        <img src={searchImg} alt='search button' />
      </NavLink>
    </>
  )
}

export default Nav;