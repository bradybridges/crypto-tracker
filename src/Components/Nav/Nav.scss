.nav-button {
  width: 33.33%;
  font-size: 1.5em;
  height: 10vh;
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: black;
  color: white;
  text-decoration: none;
}