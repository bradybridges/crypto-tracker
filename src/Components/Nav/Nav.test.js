import { shallow } from 'enzyme';
import React from 'react';
import Nav from './Nav';

describe('Nav', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Nav />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
