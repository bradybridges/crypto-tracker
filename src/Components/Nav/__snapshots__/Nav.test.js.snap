// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match the snapshot 1`] = `
<Fragment>
  <NavLink
    className="nav-button"
    to="/"
  >
    <img
      alt="home button"
      src="home.svg"
    />
  </NavLink>
  <NavLink
    className="nav-button"
    to="/portfolio"
  >
    <img
      alt="portfolio button"
      src="portfolio.svg"
    />
  </NavLink>
  <NavLink
    className="nav-button"
    to="/search"
  >
    <img
      alt="search button"
      src="search.svg"
    />
  </NavLink>
</Fragment>
`;
